cli/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ _Released 1/16/2024 (PENDING)_
 - Fixed a regression in [`13.6.2`](https://docs.cypress.io/guides/references/changelog/13.6.2) where the `body` element was not highlighted correctly in Test Replay. Fixed in [#28627](https://github.com/cypress-io/cypress/pull/28627).
 - Fixed an issue where some cross-origin logs, like assertions or cy.clock(), were getting too many dom snapshots. Fixes [#28609](https://github.com/cypress-io/cypress/issues/28609).
 - Fixed asset capture for Test Replay for requests that are routed through service workers. This addresses an issue where styles were not being applied properly in Test Replay and `cy.intercept` was not working properly for requests in this scenario. Fixes [#28516](https://github.com/cypress-io/cypress/issues/28516).
+- Fixed an issue where visiting an `http://` site would result in an infinite reload/redirect loop in recent versions of Chrome. Fixes [#25891](https://github.com/cypress-io/cypress/issues/25891).
```

Comment:
should we be more specific with the chrome version here for better posterity?

Reply:
Addressed in [`a570959`](https://github.com/cypress-io/cypress/pull/28650/commits/a570959c49267e90aee116e94fe863430e0a1f3a)

Reply:
does this also fix https://github.com/cypress-io/cypress/issues/27185?

Reply:
@emilyrohrbough That seems to be a different issue